test/test_autotokenizer.py

```diff
@@ -68,6 +68,17 @@ def test_clip_tokenizer(self):
             pre_kwargs={"WITH_DEFAULT_INPUTS": True})[0])
         actual_ids = ort_tok([text])[0]
         np.testing.assert_array_equal(ids, actual_ids)
+        
+    def test_gpt2_tokenizer(self):
```

Comment:
Does the GPT-4 tokenizer get tested by this code?

Reply:
yep - to be more precise though, the gpt4 vocab gets tested - the actual tokenization uses the same alg as the gpt2 tok which is why loading it from HF uses the HF `GPT2Tokenizer` class as well.